src/Pim/Bundle/CatalogBundle/tests/integration/PQB/Filter/Metric/LocalizableFilterIntegration.php

```diff
@@ -21,6 +21,8 @@ protected function setUp()
         parent::setUp();
 
         if (1 === self::$count || $this->getConfiguration()->isDatabasePurgedForEachTest()) {
+            $this->resetIndex();
```

Comment:
why not TestCase ? reset index is exactly like purge the database and I don't know why we should force dev to instanciate a `resetIndex()` each time they want to work with products